test/inductor/test_torchinductor.py

```diff
@@ -6288,7 +6286,6 @@ def fn():
 
                 # this can be optimized away, value too large
                 self.assertTrue("to(tl.int64)" not in code)
-                self.assertTrue("to(tl.int32)" in code)
```

Comment:
This is so nice. Triton code is going to be so much easier to read now. Forget having to track down variables that then end up being just a `tmp0 = 1`

Reply:
The issue here (and in the test above) is that they test the value range analysis. Now, with the constant folding, all these are folded. We need to change these examples so that the make sense.

Reply:
I've added a config option so we can disable constant propagation for these tests.

Reply:
I wonder whether there's anything left for ValueRanges with static shapes after constant propagation... I think there will be examples where it is still beneficial, but I couldn't name one off the top of my head. Perhaps things with `pow` and weirder ops.